blocks/columns/columns.css

```diff
@@ -37,37 +54,89 @@
     margin: var(--spacer-layout-02) auto;
 }
 
+.section-top > div:first-child {
+    margin: 0 auto;
+    padding-top: var(--spacer-layout-04);
+    }
+
+.content-band div:empty {
+    display: none;
+    }
+
+.content-band.icon-list > div > div {
+    margin-top: var(--spacer-element-05);
+    margin-bottom: var(--spacer-element-06);
+    }
+
+.columns.content-band li {
+    list-style-image: url("../icons/arrow-purple.svg");
+    margin-bottom: var(--spacer-element-03);
+    line-height: var(--line-height-160);
+    }
+
+.section.leadership-content h3 {
+    margin: var(--spacer-element-08) auto var(--spacer-element-05) auto;
+    }
```

Comment:
Why is this in `columns` block? Doesn't seem to have anything to do with columns. Is it just so that like styles are kept in the same file?

Reply:
Good question, not sure why that is here, I will move to styles.